src/content/resources/whats-new.md

```diff
@@ -18,6 +18,53 @@ and follow the [Dart blog][].
 [dart-announce]: https://groups.google.com/a/dartlang.org/d/forum/announce
 [Dart blog]: https://blog.dart.dev
 
+## 3.10 release
+
+_Released on: November 12, 2025_
+
+This section lists notable changes made from August 14, 2025,
+through November 12, 2025 to [dart.dev](https://dart.dev/docs).
+For details about the 3.10 release of Dart,
+check out the [3.10 announcement][] and the
+[3.10 SDK changelog][3-10-changelog].
+
+### Documentation updates {:.no_toc}
+
+*   Added a guide for the new [dot shorthands][] feature.
+*   Added a guide for the new [analyzer plugin][] feature.
+*   Added a guide for the [`dart build`][] tool.
+*   Added a guide for the [`dart install`][] tool.
+*   Extended the [`@Deprecated` annotation][] content with new features.
+*   Expanded the [Glossary][] with new terms including "obviously typed",
+    "closurization", "callback", "tree shaking", and "native asset".
+    (Thank you, [IldySilva](https://github.com/IldySilva))
+
+### Site improvements {:.no_toc}
+
+*   Migrated core site infrastructure to the [Jaspr framework][Jaspr migration]
```

Comment:
Consider cross linking Jaspr framework to https://jaspr.site. If you still want to link to the migration PR as well, maybe "Migrated core site infrastructure" can be.